### ISSUE TITLE
Model design and training

### ISSUE THREAD

**Issue by crn3** [crn3/vis_177249#60]
Training on PC.

UNet architecture, implemented using PyTorch.

Include Dice loss function

Monitor training, make adjustments
